slither/slither_vyper.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import os
+import subprocess
+import sys
+import vyper
+
+from slither.detectors.abstract_detector import AbstractDetector, DetectorClassification
+from slither.printers.abstract_printer import AbstractPrinter
+from .vyper_parsing.slither_vyper import SlitherVyper
+from .utils.colors import red
+
+from vyper.parser.global_context import GlobalContext
+from vyper.parser.parser import parse_to_ast
+
+logger = logging.getLogger("Slither")
+logging.basicConfig()
+
+logger_detector = logging.getLogger("Detectors")
+logger_printer = logging.getLogger("Printers")
+
+
+class Slither(SlitherVyper):
+
+    def __init__(self, filename):
+        '''
+            Args:
+                contract (str)
+            Keyword Args:
+                vyper (str): Vyper binary location (default 'vyper')
+        '''
+        self._detectors = []
+        self._printers = []
+        self._triage_mode = False
+        super(Slither, self).__init__(filename)
```

Comment:
```suggestion
        super().__init__(filename)
```
In Python 3, you just call  `super()` and the  magic happens under the hood